packages/dds/map/src/test/memory/memory_baseline.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Will there be 1 of these files per test? Or a single file with all of the test benchmarks?